### PR TITLE
compiler: run isort on compiled code

### PR DESCRIPTION
**Pull request by abn**
Split from #352 as requested by @Gobot1234

### CODE REVIEW

File: src/betterproto/plugin/compiler.py [hunk 1]
```diff
@@ -31,7 +32,20 @@ def outputfile_compiler(output_file: OutputTemplate) -> str:
     )
     template = env.get_template("template.py.j2")
 
+    code = template.render(output_file=output_file)
+    code = isort.api.sort_code_string(
+        code=code,
+        show_diff=False,
+        py_version=36,
+        profile="black",
+        force_single_line=True,
```
Comment (Gobot1234) [inline src/betterproto/plugin/compiler.py:41]: This needs turning off to match the settings in pyproject.toml

Comment (abn) [reply src/betterproto/plugin/compiler.py]: This is done.

**Comment by Gobot1234** [comment]
LGTM again. Thanks for this

**Review by Gobot1234 — APPROVED** [review]

**Comment by Gobot1234** [comment]
Sorry I didn't merge this yesterday GitHub wasn't having a good time

**Pull request merged**